### ISSUE TITLE
About the script

### ISSUE THREAD

**Issue by Lives715** [Zombie-220/Fishing#6]
Could you add a step-by-step process to use this? It might be too much to ask,

**Comment by Lives715** [comment]
this include what files to use and download and stuff

**Comment by Zombie-220** [comment]
> Could you add a step-by-step process to use this? It might be too much to ask,

I corrected the installation and use section, I hope it’s clear now

**Comment by Zombie-220** [comment]
I tried to describe the whole process from installation to fishing, it seems to me that it would be easier to upload a video guide

**Comment by Lives715** [comment]
true, thanks tho, i have a hard time visualizing texts

**Comment by Lives715** [comment]
also the first time i tried opening up the python files it would close instantly, I have phython installed, any recommendations to fix this if I cant get it to work again?

**Comment by Zombie-220** [comment]
Regarding the opening of files. Try opening it using some kind of IDE, maybe using Python IDLE (it is installed with python)

**Comment by Zombie-220** [comment]
added a video guide

**Comment by Zombie-220** [comment]
Why do you want to open a python file? If you are not going to change anything, then the dist folder will be enough for you

**Comment by Lives715** [comment]
alr, i'll use the dist folder instead

**Comment by Zombie-220** [comment]
if it’s not too difficult, then write if app works.
This is my first time writing an app for someone else

**Comment by Lives715** [comment]
are you on macos?

**Comment by Zombie-220** [comment]
no, I'm on windows. If you are on Macos, this greatly complicates the task because I don’t know how the program will behave in another OS. You will also have to run python files because .exe is not suitable for macos

**Comment by Lives715** [comment]
I'm not on macos, ig I can't get it to work then, I followed the steps

when I open the python files they go away instantly, after that, nothing seems to work..

**Comment by Lives715** [comment]
the python files are the files that make this work right?

**Comment by Zombie-220** [comment]
So.
Why doesn't your .exe start... when you run it, nothing appears, no error message?
If something appears, attach screenshots here.

Python files... if .exe does not start, you can run python files.
But first you need to install everything you need:
1. python (obviously)
2. `pip install PyQt6`
3. `pip install opencv-python`
4. `pip install PyAutoGUI`
5. `pip install keyboard`

Note: I could be wrong, but it seems everything else comes with python..
if the IDE won't run the .py and throws errors, attach them here and post them, I'll take a look

I will now download all the files of the latest version

**Comment by Dillpickilled** [comment]
I have it up and I have the buttons set to the rod and food, but for some reason when I catch the first fish, it will just keep reeling in and out. It will stop eventually but the process will repeat with the next fish.

**Comment by Zombie-220** [comment]
I tried to fix this problem in version 1.3. Please try it and let me know the results

**Comment by Dillpickilled** [comment]
I tried it out but the same issue occurs for some reason

**Comment by Zombie-220** [comment]
Did you use full screen game mode?
If you used full-screen mode and the problem remained, then I ask you to replace the images in the dist/images folder so that everything works. Also, please send it here:
1. an image that you already had in that folder
2. your cropped image
3. your screen resolution

And also thank you very much for such a quick response.

**Comment by Dillpickilled** [comment]
I didn't use full-screen mode, but I tried and it still didn't work. Also, I tried to change the image that was in the folder but I don't know how to.
Sorry for being difficult.

**Comment by Zombie-220** [comment]
You just need to replace the images with your own, when you catch a fish or something else, just take a screenshot and crop it in the likeness of existing images, and then replace them, just make sure that their names remain the same.

And I still ask you to send the above three points.

**Comment by Dillpickilled** [comment]
![Screenshot 2024-07-20 224709](https://github.com/user-attachments/assets/992ea696-0ab6-425f-9bb2-fa4fabbc38c5)
and my resolution is 1280 x 720

**Comment by Zombie-220** [comment]
do you have all 6 images in dist/images folder?
